src/base-styles/helperClasses/border/focus.css

```diff
@@ -0,0 +1,24 @@
+/*---
+title: Focus Style
+name: focusStyle
+section: Border Helpers
+---
+
+Adds standard focus style to elements
+
+class                    | description
+------------------------ | -------------------------------------------------------
+`border--focus`          | Adds standard box shadow focus style
+`border--focus--notrsn`  | Adds standard box shadow focus style without transition
+*/
+
+.border-focus:focus {
```

Comment:
name mismatch with docs - should be double hyphen (`border--focus`, `border--focus--notrsn`)